src/main.rs

```diff
@@ -72,10 +74,15 @@ async fn main() -> anyhow::Result<()> {
             (@arg APP_ID: -a --app_id +takes_value default_value("kafka_delta_ingest") 
              "The app id to use when writing to Delta.")
 
-            (@arg STARTING_OFFSETS: -o --offsets +takes_value default_value("earliest") 
-             r#"Offsets to start from. This may be 'earliest' or 'latest' to start from the earliest or latest available offsets in Kafka, 
-or a JSON string specifying the explicit offset to start from for each partition. 
-NOTE: This configuration is only applied when offsets are not already stored in delta lake."#)
+            (@arg SEEK_OFFSETS: -s --seek_offsets +takes_value
```

Comment:
For this line and https://github.com/delta-io/kafka-delta-ingest/pull/92/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR83 - I think I prefer _not_ providing a short option now so that only the long option is usable. Especially since we are changing the semantics for `-o`.

Reply:
the API is not yet finalized hence changing the sematics is allowed, but I'll remove the short option anyways